pkg/ctl/generate/profile.go

```diff
@@ -0,0 +1,84 @@
+package generate
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/git"
+	"github.com/weaveworks/eksctl/pkg/gitops"
+	"github.com/weaveworks/eksctl/pkg/gitops/fileprocessor"
+)
+
+const (
```

Comment:
You don't need parenthesis for a single constant

Reply:
Personally, I prefer to always use them outside functions, same with imports, it is just easier to read and makes package constants standout.